UiPath/Script.Activities
# OSAC-122, OSAC-129, OSAC-133 Localization Fixes for Activities

Description
Fixes for localization issues of packages
How to test
Change Studio language and check translations for activities packages
Jira
[OSAC-122](https://uipath.atlassian.net/browse/OSAC-122)
[OSAC-129](https://uipath.atlassian.net/browse/OSAC-129)
[OSAC-133](https://uipath.atlassian.net/browse/OSAC-133)
Label:
24S
